packages/frontend/routes/src/lazy.ts

```
@@ -0,0 +1,34 @@
+        }
+        return {
+          default: components[0],
+        };
+      }
+    })
+  );
+
+  return function LazyRoute(props: ComponentProps<T>) {
+    return React.createElement(Suspense, {
+      fallback,
+      children: [React.createElement(LazyComponent, props)],
```

Biome (1.9.4)

[error] Avoid passing children using a prop

The canonical way to pass children in React is to use additional arguments to React.createElement

(lint/correctness/noChildrenProp)